apps/DemoApp/DemoApp.xcodeproj/project.pbxproj

```diff
@@ -386,7 +462,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 15.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
I thought we were going to target 12?

Reply:
I think I might have overlooked this because it's just the DemoApp not the sdk deployment target but let me double check.

Reply:
Updated

Reply:
I had to leave the deployment target in DemoApp at a min of iOS 13.0 since I created the app with Xcode 13 and it uses UIScene which is only available starting from iOS 13.0. This should not be a big deal for this new sdk since starting April 25, 2022, iOS 15.0 will be required for App Store submission. But let me know if you have any concerns and I can try to create the DemoApp with the old way.